ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/WorkloadCves.helpers.js

```diff
@@ -1,8 +1,18 @@
+import { addDays, format } from 'date-fns';
+import { getDescriptionListGroup } from '../../../helpers/formHelpers';
 import { visit } from '../../../helpers/visit';
 import { selectors } from './WorkloadCves.selectors';
 
 const basePath = '/main/vulnerabilities/workload-cves/';
 
+export function getDateString(date) {
+    return format(date, 'MM/DD/YYYY');
+}
+
+export function getFutureDate(days) {
```

Comment:
What do you think about naming this `getFutureDateByDays`? I thought the added `ByDays` would make it clearer about what it's doing.